composer.json

```
@@ -13,14 +13,14 @@
 		"lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l",
 		"psalm": "psalm.phar --threads=1",
 		"psalm:update-baseline": "psalm.phar --threads=1 --update-baseline",
-		"psalm:clear": "psalm.phar --clear-cache && psalm --clear-global-cache",
+		"psalm:clear": "psalm.phar --clear-cache && psalm.phar --clear-global-cache",
 		"psalm:fix": "psalm.phar --alter --issues=InvalidReturnType,InvalidNullableReturnType,MissingParamType,InvalidFalsableReturnType",
 		"test:unit": "phpunit -c tests/phpunit.xml"
 	},
 	"require-dev": {
 		"nextcloud/coding-standard": "^1.0",
 		"psalm/phar": "^4.10",
-		"christophwurst/nextcloud": "dev-master@dev",
-		"phpunit/phpunit": "^8"
+		"phpunit/phpunit": "^8",
+		"nextcloud/ocp": "dev-stable22"
```

I think that will be necessary run `composer install` again after this change.

```suggestion
		"nextcloud/ocp": "dev-master"
```

---

This will create problems with running psalm etc. on Nextcloud 22 - 25, no?